tasks/utils/grpc.ts

```diff
@@ -31,26 +26,16 @@ export const organizationService = new OrganizationServicePromiseClient(userSvcB
 export const userService = new UserServicePromiseClient(userSvcBaseUrl)
 
 type AuthenticatedGrpcMetadata = {
-  Authorization: string,
-  'X-Organization': string
+  Authorization: string
 }
 
 const defaultOrganization = `3b25c6f5-4705-4074-9fc6-a50c28eba406`
-export const getAuthenticatedGrpcMetadata = (organizationID: string = defaultOrganization): AuthenticatedGrpcMetadata => {
-  // TODO: Implement way better API for get the current id token and DONT hardcode the organization id
-  const idToken = Cookies.get(COOKIE_ID_TOKEN_KEY)
-
-  const localStorageService = new LocalStorageService()
-  const organization = localStorageService.get<OrganizationDTO>(LOCALSTORAGE_ORGANIZATION_KEY)
-
-  if (organization) {
-    organizationID = organization.id
-  } else {
-    console.warn('Fallback to default organization')
-  }
+export const getAuthenticatedGrpcMetadata = (_: string = defaultOrganization): AuthenticatedGrpcMetadata => {
+  const token = getCurrentTokenAndUpdateInBackground()
+  return { Authorization: `Bearer ${token}` }
+}
```

Comment:
Move this method and the config for the authentication to the api package as done in https://github.com/helpwave/web/pull/1083